thoth/storages/graph/postgres.py

```diff
@@ -5001,6 +5001,48 @@ def sync_dependency_monkey_result(self, document: dict) -> None:
 
                 inspection_run.dependency_monkey_run_id = dependency_monkey_run.id
 
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str
+    ) -> List[str]:
+        """Get required symbols for a Python package in a specified version."""
+        package_name = self.normalize_python_package_name(package_name)
+        package_version = self.normalize_python_package_version(package_version)
+
+        query = self._session.query(PythonPackageVersion)
```

Comment:
Due to the new way to handle session we don't use  `self._session` but directly `session.query(...)` now.